tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -6424,21 +6423,21 @@ TEST_REQUIRE_TFLITE (testTensorFilter, propertyRank02)
   EXPECT_TRUE (gst_tensor_dimension_string_is_equal (input_dim, "3:224:224:1"));
   g_free (input_dim);
 
-  /* Rank should be 3 since input dimension string is not given. */
+  /* Rank should be 4 since input dimension string is not given. */
   gchar *input_ranks;
   g_object_get (filter, "inputranks", &input_ranks, NULL);
-  EXPECT_STREQ (input_ranks, "3");
+  EXPECT_STREQ (input_ranks, "4");
   g_free (input_ranks);
 
   gchar *output_dim;
   g_object_get (filter, "output", &output_dim, NULL);
   EXPECT_TRUE (gst_tensor_dimension_string_is_equal (output_dim, "1001:1:1:1"));
```

Comment:
May change `"1001:1:1:1"` to `"1001:1"` for consistency.
Later, will those two dim strings become not equal?

Reply:
Changed to `"1001:1"` :)
`"1001:1"` is saved like {1001,1,0,0} and treated like `"1001:1:1:1"`.
So two dim strings will become equal

Reply:
Thanks!